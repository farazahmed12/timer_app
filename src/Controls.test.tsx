import React from "react";
const TestRenderer = require("react-test-renderer");

import Controls from "./Controls";

it("renders when there are no items", () => {
  const tree = TestRenderer.create(<Controls />).toJSON();
  expect(tree).toMatchSnapshot();
});
