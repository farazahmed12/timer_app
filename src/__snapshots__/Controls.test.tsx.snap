// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there are no items 1`] = `
<div
  className="controls-container"
>
  <button
    onClick={[Function]}
  >
    Play
  </button>
  <button
    onClick={[Function]}
  >
    Stop
  </button>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
</div>
`;
